packages/generator/gulpfiles/downloadRamlsTask.ts

```diff
@@ -93,13 +91,30 @@ function downloadRamlFromExchange(): Promise<void> {
   });
 }
 
-gulp.task("cleanInputDir", async () => {
-  return fs.removeSync(path.join(config.inputDir));
+gulp.task("diffRamlFiles", async () => {
+  const result = await diffNewAndArchivedRamlFiles(
+    config.apiBackupDir,
+    config.inputDir,
+    path.join(config.apiBackupDir, config.apiConfigFile),
+    path.join(config.inputDir, config.apiConfigFile)
+  );
+  return fs.writeFile(config.diffFile, JSON.stringify(result));
+});
+
+gulp.task("cleanup", async () => {
```

Comment:
```suggestion
gulp.task("cleanup", () => {
```
Either this or use async `fs`.

Reply:
I already tried that, gulp doesn't like it. It complains about missing `async`.

Reply:
Gulp tasks must be async.  They always return a promise.  What gaurav is doing here is fine.